trl/trainer/ppo_trainer.py

```diff
@@ -937,6 +939,8 @@ def batched_forward_pass(
         all_masks = []
         all_values = []
 
+        model.eval()
```

Comment:
Here I'm following the same logic of `transformers.Trainer` to put the model in eval model during inference - this is needed to ensure the KL divergence is 0 at step 0 with ZeRO-3